lte/gateway/python/magma/subscriberdb/protocols/s6a_proxy_servicer.py

```diff
@@ -13,12 +13,14 @@
 
 import logging
 
-from feg.protos import s6a_proxy_pb2, s6a_proxy_pb2_grpc
-from google.protobuf.json_format import MessageToJson
 from magma.subscriberdb import metrics
 from magma.subscriberdb.crypto.utils import CryptoError
 from magma.subscriberdb.store.base import SubscriberNotFoundError
 
+from feg.protos import s6a_proxy_pb2, s6a_proxy_pb2_grpc
```

Comment:
There is no need to make any changes to this file. All the 5G authentication services are moved to its own services.

Reply:
reverted all changes done in this file.